README.md

```diff
@@ -42,6 +42,33 @@ The services of a client divide the API into logical chunks and correspond to
 the structure of the GitHub API documentation at
 https://developer.github.com/v3/.
 
+
+A basic unauthenticated program might look something like:
+
+```go
+import (
+	"context"
+	"github.com/google/go-github/github"
+)
+
+func FetchOrganizations(username string) ([]*Authorization, error) {
+	ctx := context.Background()
+	client := github.NewClient(nil)
+
+	// list all organizations for user "willnorris"
+	orgs, _, err := client.Organizations.List(ctx, "willnorris", nil)
+
+	if err != nil {
+		return nil, err
+	}
+	return orgs, err
+}
+```
+
+For more sample code snippets, head over to the
+[examples](https://github.com/google/go-github/tree/master/example) directory.
```

Comment:
The directory is named `example`, not `examples`, you should refer to it accordingly. ([This](https://dmitri.shuralyov.com/idiomatic-go#use-singular-form-for-collection-repo-folder-name) explains why.)

Reply:
Thanks a lot for the resource @shurcooL 
I will make the necessary changes.

I read through the [Contributing Guide](https://github.com/google/go-github/blob/master/CONTRIBUTING.md) and I had a question regarding pushing a new commit(with the desired changes) so as to update the PR.

Should I push both the commits or should I squash them into a single commit on my end?

Reply:
We should add a mention for it there, but for a contributor, you should prefer to always push separate, new commits to PRs, rather than squashing and force pushing existing ones.

The reason to do so is because it makes reviewing the PR easier. Each change can be seen separately, so once a PR is reviewed partially, only the new changes need to be reviewed, rather than starting from scratch each time.

When we merge your PR, we will squash all your commits into a single clean logical commit.